src/components/Layout/Footer.tsx

```diff
@@ -252,7 +252,8 @@ const Footer: React.FC<React.HTMLProps<HTMLDivElement>> = (props) => {
           className={styles.copyright}
         >
           <p>
-            &copy; 2012-
+            &copy;
+            {' '}
             {new Date().getFullYear()}
             {' '}
             MaxMind, Inc. All Rights Reserved.
```

Comment:
I think we can drop the "All Rights Reserved." bit as well.

Reply:
It's not a big deal.  I could go either way.

Reply:
It is possible that Diana would want us to add that trademark language. I am not sure if that changes anything, i.e.:

> MaxMind, GeoIP, minFraud, and related trademarks belong to MaxMind, Inc.

Reply:
I asked about that in the Slack thread.

Reply:
Oh, I see it on the dev site already. Sorry, I meant only remove the "All Rights Reserved." bit, which is for copyright. We want the trademark language presumably.